drivers/include/net/netdev2.h

```diff
@@ -74,6 +74,9 @@ typedef enum {
     NETDEV2_EVENT_RX_COMPLETE,      /**< finished receiving a packet */
     NETDEV2_EVENT_TX_STARTED,       /**< started to transfer a packet */
     NETDEV2_EVENT_TX_COMPLETE,      /**< finished transferring packet */
+#ifdef MODULE_OPENTHREAD
+    NETDEV2_EVENT_TX_COMPLETE_DATA_PENDING, /**< finished transferring packet and has data pending flag **/
```

Comment:
Okay, then let's make this a new PR independent of OpenThread! But might I suggest a different name and doc? I'm not sure what "finished transferring packet and has data pending flag" from a high-level stand-point means. But how about the following?

```C
    NETDEV2_EVENT_TX_PENDING,  /**< data was partly transmitted successfully, but more data is pending */
```